README.md

```diff
@@ -126,6 +129,14 @@ assertDisplayed(R.string.hello_world);
 assertDisplayed(R.id.button);
 assertDisplayed(R.id.button, "Hello world")
 
+// Will keep checking until either the view is found or the timeout duration has elaspsed
+assertDisplayedAfterWait(R.id.button);
```

Comment:
I'm not sure about this. It goes against Espresso way of waiting for things, which is by using IdlingResources. You should not manually need to wait for a view or use sleeps.